ivy.xml

```diff
@@ -11,7 +11,8 @@
   <dependency org="javax.servlet" name="javax.servlet-api" rev="3.1.0" />
   <dependency org="javax.mail" name="mail" rev="1.4.5" />
   <dependency org="org.dom4j" name="dom4j" rev="${dom4j.version}" />
-  <dependency org="log4j" name="log4j" rev="1.2.16" />
+  <dependency org="org.apache.logging.log4j" name="log4j-core" rev="${log4j.core.version}" />
```

Comment:
I guess client side zimlet code should not depend on log4j libs, we can remove this